example-app/SantokuApp/jest/__mocks__/expo-secure-store.ts

```diff
@@ -3,9 +3,11 @@ export const getItemAsync = jest.fn(() => '1234567890');
 export const deleteItemAsync = jest.fn();
 export const isAvailableAsync = jest.fn();
 
-// expo-secure-storeのすべてのNamed Exportを列挙するのは大変なので、
-// ES6のexport/import形式ではなく、module.exportsを使ってexportする。
-// eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
+/*
+  eslint-disable-next-line @typescript-eslint/no-unsafe-assignment --
+  @react-navigation/nativeのすべてのNamed Exportを列挙するのは大変なので、
```

Comment:
`example-app/SantokuApp/jest/__mocks__/expo-secure-store.ts`
> @react-navigation/nativeのすべてのNamed Exportを列挙するのは大変なので、

fix コピペミス
```suggestion
  expo-secure-storeのすべてのNamed Exportを列挙するのは大変なので、
```